frontend/apps/core-app/src/screens/RecipientRegistrationScreen/RecipientRegistrationScreen.container.tsx

```diff
@@ -23,12 +274,16 @@ export const RecipientRegistrationScreenContainer: React.FC = () => {
     setMode('register');
   }, [JSON.stringify(forms)]);
 
+  const handleSubmit = (data: any) => {
+    console.log(data);
+  };
+
   if (mode === 'register') {
     return (
       <RecipientRegistrationScreenComponent
         forms={forms}
         records={records}
-        onSubmit={() => {}}
+        onSubmit={handleSubmit}
```

Comment:
this will be changed to just switch to review mode, right?

Reply:
Yeah all this will change and was just used to console log the values